django/README.md

```diff
@@ -1 +1,7 @@
 # Django
+
+## Table of Contents
+
+- [Model](django/models/README.md)
+- [Project Structure](django/project-structure/README.md)
+- [View Sets](django/viewsets/README.md)
```

Comment:
ViewSets?

Reply:
I think correct grammar is View Sets krub since it's two words, but will change krub so it match the Django Rest Doc